providers/view_provider.ts

```diff
@@ -0,0 +1,152 @@
+/*
+ * @adonisjs/view
+ *
+ * (c) AdonisJS
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+import { ApplicationService, HttpRouterService } from '@adonisjs/core/types'
+import { ViewContract } from '../types/index.js'
+import { BriskRoute, HttpContext } from '@adonisjs/core/http'
+
+/**
+ * View provider to register view to the application
+ */
+export default class ViewProvider {
+  constructor(protected app: ApplicationService) {}
+
+  /**
+   * Add globals for resolving routes
+   */
+  private addRouteGlobal(view: ViewContract, router: HttpRouterService) {
```

Comment:
Not using the `private` modifier anymore. Instead using `#` for creating JavaScript private properties